lttoolbox/fst_processor.cc

```diff
@@ -1400,25 +1101,25 @@ FSTProcessor::initBiltrans()
 }
 
 
-wstring
-FSTProcessor::compoundAnalysis(wstring input_word, bool uppercase, bool firstupper)
+UString
+FSTProcessor::compoundAnalysis(UString input_word, bool uppercase, bool firstupper)
 {
   const int MAX_COMBINATIONS = 32767;
 
   State current_state = initial_state;
 
   for(unsigned int i=0; i<input_word.size(); i++)
   {
-    wchar_t val=input_word.at(i);
+    UChar val=input_word.at(i);
```

Comment:
`.at()`? There should be no `.at()` in the codebase. It's already bounds checked in the `for()` loop, so `[]` is fine and faster.